infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -197,27 +203,31 @@ public void generateConfig(Set<LogReplicationSession> sessions) {
                     case LOGICAL_GROUPS:
                         log.debug("Generating LOGICAL_GROUP config for session {}",
                                 TextFormat.shortDebugString(session));
-                        generateLogicalGroupConfig(session);
+                        generateLogicalGroupConfig(session, registryTableOnly);
                         break;
                     default: break;
```

Comment:
let us throw an exception here

Reply:
yes it's an unexpected path, but instead of throwing an exception I tend to just leave a log msg here. As this method is generating config for multiple sessions, in error case the exception will just terminate the execution. 

and more importantly I think is error path is very hard to trigger, as the `ReplicationModel` field is controlled by our code (in the clients or the default FULL_TABLE one)